array-methods/2-array.js

```diff
@@ -1,45 +1,79 @@
 var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
 ];
+
 //Write a function findLongestWord that takes an array of words and returns the longest one.
 //If there are 2 with the same length, it should return the first occurrence.
 
+let longestWord = words.reduce((acc, cv) => {
+  // console.log(acc +" "+ cv);
+  if (acc.length < cv.length) {
+    acc = cv;
+  }
+  return acc;
+});
 
+console.log(longestWord); //crocodile
 
-
+//==
 var numbers1 = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 // Create a sumArray function that takes an array of numbers1 as a parameter, and calculate the sum of all its numbers
 // Use reduce method of array
 // Use the above sum and calculate the average.
 
+var sum = numbers1.reduce((acc, cv) => {
+  acc += cv;
+  return acc;
+}, (acc = 0));
```

Comment:
You don't have to say `acc = 0` you can just pass `0` as the second parameter and it should work.